tests/core/test_channels.py

```diff
@@ -911,3 +911,210 @@ def test_has_user_permission_to_send_messages_to_conversation_without_permission
     assert not RasaChatInput._has_user_permission_to_send_messages_to_conversation(
         jwt, message
     )
+
+
+def test_hangouts_channel():
```

Comment:
Thanks a lot for adding tests!

One optional thing: could you please move hangouts related tests to tests/core/channels/hangouts.py?
It's one big test file now and it becomes quite difficult to follow.

Reply:
Done that